<launch>
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find kuka_pose_following)/config/logger.conf"/>
  <node pkg="kuka_pose_following" type="pose_following" name="pose_following" output="screen">
    <rosparam>
      rate: 40
      group: 'kuka_full'
      w0: 0.01
      k0: 0.01
      theta_d_lim: 3.14
      dt_pose_lim: 0.5
    </rosparam>
  </node>

  <node pkg="kuka_pose_following" name="posefollowing_client" type="posefollowing_client" />
  <node pkg="kuka_pose_following" type="moveit_interface" name="moveit_interface" output="screen"/>

</launch>
